doc/src/read_dump.rst

```diff
@@ -230,13 +231,27 @@ will then have a label corresponding to the fix-ID rather than "x" or
 "xs".  The *label* keyword can also be used to specify new column
 labels for fields *id* and *type*\ .
 
-For dump files in *xyz* format, only the *x*, *y*, and *z* fields are
-supported.  The dump file does not store atom IDs, so these are
-assigned consecutively to the atoms as they appear in the dump file,
-starting from 1.  Thus you should ensure that order of atoms is
-consistent from snapshot to snapshot in the XYZ dump file.  See
-the :doc:`dump_modify sort <dump_modify>` command if the XYZ dump file
-was written by LAMMPS.
+For dump files in *xyz* format, only the *type*, *x*, *y*, and *z*
+fields are supported.  There are many variants of the XYZ file format.
+LAMMPS will read the number of atoms from the first line of each frame,
+ignore the second (title) line, and then read one line for each atom in the format:
+
+.. parsed-literal::
+
+   <label> <x coordinate>  <y coordinate> <z coordinate>
+
+
+If label is an integer number as label (like with XYZ files created by
+created with default settings by :doc:`dump style <dump>` *xyz*), that
+number will be used at atom type.  If the atom label is a string, then
+it is required that a map is created using the :doc:`labelmap command
+<labelmap>`. This map needs to associate each (numeric) atom type with
+one string label and that numeric type is stored.  The xyz format dump
```

Comment:
new paragraph starting: The xyz format...

Reply:
all done.